### PR TITLE
Add more domains

### PR DESCRIPTION
**Pull request by ToutyRater**

### CODE REVIEW

File: data/baidu [hunk 1]
```diff
@@ -6,8 +6,13 @@ baidustatic.com
 baifubao.com
 bdimg.com
 bdstatic.com
+bdydns.net
+dlnel.com
 duapps.com
+jomocdn.net
+jomocdn.net
```
Comment (iochen) [inline data/baidu]: duplicate with the domain in `data/baidu#L12`

Comment (ToutyRater) [reply data/baidu]: Solved

**Comment by iochen** [comment]
file `jiguang` is in a wrong path

**Comment by Loyalsoldier** [comment]
Fixed.

**Pull request merged**